### PR TITLE
Feature css value control branch

### PR DESCRIPTION
**Pull request by natlusyht**
This is a branch of [feature](https://github.com/webstudio-is/webstudio-designer/pull/313) with only text-control reverted to it's previous state.

The issue I mentioned in https://github.com/webstudio-is/webstudio-designer/pull/268/files#r979314152 is still present though.

## Before requesting a review

- [ ] if the PR is WIP - use draft mode
- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")
- [ ] what kind of review is needed?
  - [ ] conceptual
  - [ ] detailed
  - [ ] with testing

## Test cases

- [ ] step by step interaction description and what is expected to happen

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

### CODE REVIEW

File: apps/designer/app/designer/features/style-panel/shared/property-name.tsx [hunk 1]
```diff
@@ -58,6 +58,11 @@ export const PropertyIcon = forwardRef<
         ref={forwardRef}
         size="1"
         css={{
+          // @note DO NOT REMOVE WITHOUT REPLACING WITH TOKENS
```
Comment (kof) [inline apps/designer/app/designer/features/style-panel/shared/property-name.tsx:61]: The idea Andrey and I are pushing is to actually not do the nudges at all and instead fix it later properly with changes in the scales etc, so the preference is towards have small suboptimal visual things instead of heaving these

File: apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx [hunk 2]
```diff
@@ -15,37 +39,195 @@ export const TextControl = ({
...
-        type: "keyword",
-        value: item.name,
-      }))}
-      onChange={(value) => {
-        console.log("change", value);
-        handleChange(value, true);
+    <Combobox
+      name={styleConfig.property}
+      items={items}
+      value={String(value.value)}
```
Comment (kof) [inline apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:101]: we discsussed that combobox should receive the value object, not the value.value string, why is it back?

File: apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx [hunk 3]
```diff
@@ -15,37 +39,195 @@ export const TextControl = ({
...
+              suffix={
+                <IconButton
+                  {...(value.type !== "unit" && toggleProps)}
+                  css={{
+                    visibility: items.length ? "visible" : "hidden",
+                    position: "absolute",
+                    right: "1px",
+                    top: "1px",
+                    width: "auto",
+                    height: "calc(100% - 2px)",
```
Comment (kof) [inline apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:173]: again, we said no nudges

Comment (kof) [reply apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:173]: positioning should be done via layout
sizes should be done using token values, if there is none that fits the exact pixel value we need - take the next closest, we have to fix tokens, not workaround with nudges

File: apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx (hunk 2 above)
Comment (kof) [reply apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:101]: of course it doesn't, you have to pass to the input the value.value

Comment (okonet) [reply apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:101]: I think you must define `itemToString` to make it work. This will also fix filtering.

Comment (okonet) [reply apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:101]: I think I'll make it a required prop so it removes this confusion.

Comment (kof) [reply apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:101]: \when rendering the value in the input, he can just pass value.value, its in his control where TextField is getting rendered

Comment (okonet) [reply apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx:101]: I'm confused. What is "that"? It was working in my branch. Maybe take a look there? Just a heads up: I'm working on some combo box changes in the original branch to 1. Make custom usages easier and 2. To revert the input and prefix behavior I introduced earlier.

File: packages/design-system/src/components/primitives/numeric-gesture-control.ts [hunk 4]
```diff
@@ -90,6 +90,8 @@ export const numericScrubControl = (
         break;
       }
       case "pointerdown": {
+        // light touches don't register corresponding pointerup
+        if (event.pressure === 0) break;
```
Comment (kof) [inline packages/design-system/src/components/primitives/numeric-gesture-control.ts:94]: @natlusyht will send this in a separate pr

**Comment by kof** [comment]
Closed in favor of https://github.com/webstudio-is/webstudio-designer/pull/313
